_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_utils.cpp

```diff
@@ -2377,7 +2377,11 @@ mfxStatus MfxVideoParam::GetSliceHeader(Task const & task, Task const & prevTask
                     }
 
                     if (task.m_ldb && Equal(task.m_refPicList[0], task.m_refPicList[1]))
-                        Copy(s.pwt[1], s.pwt[0]);
+                    {
```

Comment:
do we still need old Copy() or definition can be dropped?

Reply:
@dvrogozh, It's probably dropped because compilation failed with old implementation. That's why I've changed the code. Before this code wasn't compiled due to disabled define in mfx_config.h